src/middleware/attach-middleware.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import BaseMiddleware from './base-middleware';
+
+/**
+ * Middleware implementation for bumper plugin.
+ * @classdesc
+ */
+class AttachMiddleware extends BaseMiddleware {
+  id: string = 'AttachMiddleware';
+  _context: Player;
+
+  /**
+   * @constructor
+   * @param {Player} context - The Player context.
+   */
+  constructor(context: Player) {
+    super();
+    this._context = context;
+  }
+
+  /**
+   * Play middleware handler.
+   * @param {Function} next - The next play handler in the middleware chain.
+   * @returns {void}
+   */
+  play(next: Function): void {
+    if (!this._context.src) {
+      this._context._load();
```

Comment:
This change actually re-open FEC-9459 issue (play bypasses the `load` middleware mechanism).
Consider plugin that has `load` middleware that postpones the player loading by timeout.
By this, the loading will occur immediately.